build.gradle

```diff
@@ -143,7 +95,9 @@ ext {
       "--add-opens=java.security.jgss/sun.security.krb5=ALL-UNNAMED"
     )
 
-  userMaxForks = project.hasProperty('maxParallelForks') ? maxParallelForks.toInteger() : null
+  maxTestForks = project.hasProperty('maxParallelForks') ? maxParallelForks.toInteger() : Runtime.runtime.availableProcessors()
+  maxScalacThreads = project.hasProperty('maxScalacThreads') ? maxScalacParallelism.toInteger() :
+      Math.min(Runtime.runtime.availableProcessors(), 8)
```

Comment:
You probably need the mavenUrl, mavenPassword, mavenUsername changes to workaround KC-1802. I shared an internal link privately with you.

Reply:
Thanks for the review.  I pushed a new commit to update maven properties.